### PR TITLE
Contact form of Portofolio

### PR DESCRIPTION
**Pull request by SabaAhmad404**
This is a finished portfolio mobile website. In this project, i work with my coding partner to create a contact form of the portofolio  In this project 
1) added linters
2) added HTML and CSS
kindly review my project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -196,8 +196,26 @@ <h4>LET'S  CONNECT</h4>
...
-              </section> 
+              </section>
+              <section class="contact-area">
+                <div class="contact-me">
+                   <h2 class="contact-title">Contact Me</h2>
+                   <p class="contact-desc">
+                       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+                   </p>
+                   
+                           <form action="contact-form" method="post">
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! Your `Form` should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html). You have used `action` and `method` in form but you made some mistake in value given to `action`, Please use formspree to give value to `action`!  `[HINT]` The `action` value should be similar to `https://formspree.io/f/{form_id}`

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @SabaAhmad786, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -196,8 +196,26 @@ <h4>LET'S  CONNECT</h4>
...
-              </section> 
+              </section>
+              <section class="contact-area">
+                <div class="contact-me">
+                   <h2 class="contact-title">Contact Me</h2>
+                   <p class="contact-desc">
+                       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+                   </p>
+                   
+                           <form action="https://formspree.io/forms" method="post">
```
Comment (aimalamiri) [inline index.html]: Good job, but as the previous reviewer requested, please make sure to use a proper form link. For this visit the [Formspree](https://formspree.io/html) website and create an account. You'll be able to create a form once you log in.

It throws an error currently for the incorrect form link.

image

File: index.html [hunk 3]
```diff
@@ -196,8 +196,26 @@ <h4>LET'S  CONNECT</h4>
             </ul>
             </div>
            </div>
-              </section> 
+              </section>
```
Comment (aimalamiri) [inline index.html:199]: [Optional] In order to make your webpage look better, it's important to follow the Figma template. So please try to remove the extra space from the right.

image

File: index.html [hunk 4]
```diff
@@ -196,8 +196,26 @@ <h4>LET'S  CONNECT</h4>
...
+              <section class="contact-area">
+                <div class="contact-me">
+                   <h2 class="contact-title">Contact Me</h2>
+                   <p class="contact-desc">
+                       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
```
Comment (aimalamiri) [inline index.html]: [Optional] It looks great, but it's important to follow the Figma template. So please try to follow the same Figma font size.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @SabaAhmad786, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereReachGIF


 ## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -344,8 +342,147 @@ h5 {
...
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+  margin-top: 22px;
+}
+
+.contact-area {
+  background-color: #fff;
+  padding: 16px 20px 16px 0;
+}
```
Comment (levy002) [inline style.css:442]: - [OPTIONAL] I highly recommend removing the padding on this container in order to look like the provided Figma design. :wink: 

    - ### Your design (with padding)

   image

    - ### Your design (without padding)

   image

**Review by levy002 — APPROVED** [review]
Hi @SabaAhmad786 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 💯🌟 

5SW

## Status: APPROVED 🔥 🔥 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**